src/internal/data/item_test.go

```diff
@@ -146,9 +146,15 @@ func (mid *mockItemDataGetter) check(t *testing.T, expectCalled bool) {
 }
 
 func (mid *mockItemDataGetter) GetData(
-	context.Context,
+	ctx context.Context,
+	errs *fault.Bus,
 ) (io.ReadCloser, *details.ItemInfo, bool, error) {
 	mid.getCalled = true
+
+	if mid.err != nil {
+		errs.AddRecoverable(ctx, mid.err)
+	}
```

Comment:
I'm gonna let @ryanfkeepers take a look at this one. I'm not sure if we have to add it the recoverable list here. Should this maybe in `lazyItem.ToReader`?

Reply:
Since this is a mock, it should be fine.